omnibus-base-sepolia-andromeda.toml

```diff
@@ -10,12 +10,10 @@ include = [
     "tomls/pools/spartan-council.toml",                             # Create Spartan Council Pool
     "tomls/markets/spot-factory.toml",
     "tomls/markets/perps-highcap-factory.toml",                     # First supermarket (High MCap)
-    "tomls/markets/perps-lowcap-factory.toml",                      # Second supermarket (Low MCap)
     "tomls/markets/common/bigcap-settings.toml",
     "tomls/omnibus-base-sepolia-andromeda/spot/usdc.toml",          # sUSDC spot market/wrapper, mock for USDC
     "tomls/omnibus-base-sepolia-andromeda/spot/statausdc.toml",     # sStataUSDC spot market/wrapper, mock for aUSDC/stataUSDC from erc-4626-to-assets-ratio-oracle
     "tomls/permit-all-perps-perpsHighcapSystem.toml",               # Enable perps market
-    "tomls/permit-all-perps-perpsLowcapSystem.toml",                # Enable perps market
```

Comment:
Need rug access to this market for everyone, not just remove "allowAll"

Reply:
it's fine because it's sepolia, right?

Reply:
fair enough, not a big deal yeah

Reply:
already fixed in the previous PR